src/styles/_catalog.scss

```diff
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
   margin: 0 auto;
+  gap: 48px;
+
+  @media (max-width: 1023px) {
```

Comment:
Let's create mixin for breakpoints, usually we use mixins for common breakpoints in media queries